FindmallocMC.cmake

```diff
@@ -96,8 +93,14 @@ if(mallocMC_ROOT_DIR)
     string(REGEX MATCH "([0-9]+)" mallocMC_VERSION_PATCH
                                 ${mallocMC_VERSION_PATCH_HPP})
 
+    # mallocMC variables ########################################################
+    #
+    set(mallocMC_REQUIRED_VARS_LIST mallocMC_ROOT_DIR ; mallocMC_INCLUDE_DIRS)
```

Comment:
oh it's as I said [see docs for `list`](http://www.cmake.org/cmake/help/v3.0/command/list.html):
```
NOTES: A list in cmake is a ; separated group of strings. To create a list the set command can be used. For example, set(var a b c d e) creates a list with a;b;c;d;e, and set(var “a b c d e”) creates a string or a list with one item in it. (Note macro arguments are not variables, and therefore cannot be used in LIST commands.)
```